### ISSUE TITLE
How make crops?

### ISSUE THREAD

**Issue by enoter** [Jukiro/Botania-PE#1]
You can write example with custom crops?)

**Comment by Jukiro** [comment]
Yeah, you can tessellate it with TileTessellator or set it's renderType
with the renderTypeId of crops...

**Comment by enoter** [comment]
I not particularly understand, but do you mean to do in your addon?

**Comment by Jukiro** [comment]
Yeah. I think you can do it ModPE! :)

**Comment by enoter** [comment]
ModPE shit and adding in MCPE more lags

**Comment by ManassehPierce** [comment]
Use TileTessellator::tessellateRowInWorld();
and
TileTessellator::tessellateRowTexture();
for crops.

**Comment by Jukiro** [comment]
You can also make your own renderer on TileTessellator.
